Code/client/Services/Generic/CharacterService.cpp

```diff
@@ -493,6 +513,14 @@ void CharacterService::OnCharacterSpawn(const CharacterSpawnRequest& acMessage)
     if (pActor->IsDead() != acMessage.IsDead)
         acMessage.IsDead ? pActor->Kill() : pActor->Respawn();
 
+    spdlog::info("Spawn Reqeust Is summon {} {}", acMessage.IsPlayerSummon, m_world.playerHandler ? "handle" : "no handle");
+
+    if (acMessage.IsPlayerSummon && m_world.playerHandler)
+    {
+        pActor->SetCommandingActor(m_world.playerHandler);
+        m_world.GetRunner().Trigger(MoveActorEvent(pActor->formID, pActor->GetParentCell()->formID, pActor->position));
```

Comment:
Why move it to itself?

Reply:
The Actor won't actually become a real CommandedActor until you trigger the MoveActorEvent, unsure why. I'm guessing it has something to do with updating the actor's state. Will be removed onces I've found a better method.